### PR TITLE
Upgrade to Rails 5.2

### PR DESCRIPTION
**Pull request by sasharevzin**
~Still working on it. Please, hold with reviews because I'm planning to break into the logical commits~
I added some files that come by default with Rails 5. They can be used for future configurations.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,17 +1,19 @@
-source 'http://rubygems.org'
+source 'https://rubygems.org'
 
 ruby '2.4.9'
```
Comment (evadne) [inline Gemfile:3]: This may be unrelated, but we have newer Ruby versions now.

Support for Ruby 2.4 has ended!
https://www.ruby-lang.org/en/news/2020/04/05/support-of-ruby-2-4-has-ended/

File: config/initializers/secret_token.rb [hunk 2]
```diff
@@ -6,7 +6,5 @@
 # If you change this key, all old signed cookies will become invalid!
 # Make sure the secret is at least 30 characters and all random,
 # no regular words or you'll be exposed to dictionary attacks.
-Safecast::Application.config.secret_token =
```
Comment (evadne) [inline config/initializers/secret_token.rb:9]: Why was this changed?

**Review by evadne — CHANGES_REQUESTED** [review]
- the `secret_token` change may break the app, so it is best to find out why it was changed.

File: Gemfile (hunk 1 above)
Comment (sasharevzin) [reply Gemfile:3]: Little by little. Let's finish with Rails ;)

File: config/initializers/secret_token.rb (hunk 2 above)
Comment (sasharevzin) [reply config/initializers/secret_token.rb:9]: Deprecated: https://stackoverflow.com/questions/55451394/cannot-understand-rails-deprecation-warning-for-secret-token
As long as we use same key, we should be fine, but of course, we'll testing first. 
Btw, is there staging env?

Comment (evadne) [reply config/initializers/secret_token.rb:9]: 1. The problem is that the values are different in the diff which is why I flagged it.

2. Staging environment: As far as I recall, yes, Rob Oudendijk and Mat Schaffer should know.

File: Gemfile (hunk 1 above)
Comment (evadne) [reply Gemfile:3]: 👌

**Comment by sasharevzin** [comment]
Closing in favor of https://github.com/Safecast/safecastapi/pull/658
